mesonbuild/minstall.py

```diff
@@ -89,6 +89,44 @@ def sanitize_permissions(path, umask):
     except PermissionError as e:
         msg = '{!r}: Unable to set permissions {!r}: {}, ignoring...'
         print(msg.format(path, new_perms, e.strerror))
+    except FileNotFoundError as e:
```

Comment:
As said, maybe never do this for symlinks at all?

Reply:
Ah, right I can do that too... the idea here was i wasn't aware of symlinks having different permissions too, but so it seems possible in some platforms, so let's go with that indeed.

Reply:
Ops, not... So, `NotImplementedError: chmod: follow_symlinks unavailable on this platform` will happen, thus we should ignore that too.

Reply:
Yes, looks like it makes more sense to ignore os.chmod/chown completely for symlinks.